tests/bsim/bluetooth/host/gatt/sc_indicate/src/main.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bstests.h"
+#include "bs_bt_utils.h"
+
+#define BS_SECONDS(dur_sec)    ((bs_time_t)dur_sec * 1000000)
```

Comment:
```suggestion
#define BS_SECONDS(dur_sec)    ((bs_time_t)dur_sec * USEC_PER_SEC)
```
Also, BS_SECONDS isn't a great name for something that's microseconds